docs/reference/connector/apis/update-connector-api-key-id-api.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[update-connector-api-key-id-api]]
+=== Update connector API key id API
+++++
+<titleabbrev>Update connector API key id</titleabbrev>
+++++
+
+preview::[]
+
+Updates the `api_key_id` and/or `api_key_secret_id` field(s) of a connector, specifying:
+
+. The id of the API key used for authorization
+. The id of the Connector Secret where the API key is stored
```

Comment:
Might be helpful to give a little handholding about where to find this or link to relevant doc

Reply:
We didn't write docs for any secrets storage stuff because the connector secrets API was supposed to be internal. Maybe that has changed and we need docs now, since we've so much documentation around programmatically. creating native connectors. WDYT?

Reply:
Yeah not sure really, but we want users who read this to know _how_ to find the Connector Secret id

Reply:
Would a link to the Enterprise Search docs with programmatic steps for creating a native connector be enough?

Reply:
cc @seanstory for input

Reply:
I think we want a programmatic story for native and client connectors. If we can't programmatically create and run native connectors without secrets apis, I think we'll need to document the secrets APIs. 

> Would a link to the Enterprise Search docs with programmatic steps for creating a native connector be enough?

That sounds reasonable to me.

Reply:
@leemthompo I've added a section for this under the dotpoints

Reply:
I love succinct 😄